### PR TITLE
Ecotone spec updates for system_config

### PR DESCRIPTION
**Pull request by roberto-bayardo**
**Description**

Account for Ecotone upgrade in the system-config spec.

### CODE REVIEW

**Comment by roberto-bayardo** [comment]
cc: @anikaraghu

File: specs/system_config.md [hunk 1]
```diff
@@ -89,6 +93,7 @@ The contained log events are filtered and processed as follows:
   - type `1`: `overhead` and `scalar` overwrite, as two packed `uint256` entries.
   - type `2`: `gasLimit` overwrite, as `uint64` payload.
   - type `3`: `unsafeBlockSigner` overwrite, as `address` payload.
+  - type `4`: `l1BasefeeScalar` and `l1BlobBasefeeScalar` overwrite, as two packed `uint32` entries.
```
Comment (tynes) [inline specs/system_config.md]: As two packed as in `abi.encodePacked` rather than `abi.encode`?

File: specs/system_config.md [hunk 2]
```diff
@@ -31,10 +31,12 @@ Version `0` embeds the current batch submitter ethereum address (`bytes20`) in t
 In the future this versioned hash may become a commitment to a more extensive configuration,
 to enable more extensive redundancy and/or rotation configurations.
 
-### `overhead` and `scalar` (`uint256,uint256`)
+### `overhead`,`scalar` (`uint256,uint256`) and `l1BasefeeScalar`,`l1BlobBasefeeScalar` (`uint32,uint32`)
 
-The L1 fee parameters, also known as Gas Price Oracle (GPO) parameters,
-are updated in conjunction and apply new L1 costs to the L2 transactions.
+These parameters are the L1 fee parameters, also known as Gas Price Oracle (GPO) parameters, used
+to compute the L1 data fee applied to an L2 transaction.  Prior to the Eclipse upgrade, `overhead`
```
Comment (tynes) [inline specs/system_config.md]: nit: this still says eclipse

File: specs/system_config.md (hunk 1 above)
Comment (roberto-bayardo) [reply specs/system_config.md]: clarified should be in abi.encodePacked() format.

File: specs/system_config.md [hunk 3]
```diff
@@ -70,7 +72,9 @@ A rollup node initializes its derivation process by finding a starting point bas
 - When started from L2 genesis, the initial system configuration is retrieved from the rollup chain configuration.
 - When started from an existing L2 chain, a previously included L1 block is determined as derivation starting point,
   and the system config can thus be retrieved from the last L2 block that referenced the L1 block as L1 origin:
-  - `batcherHash`, `overhead` and `scalar` are retrieved from the L1 block info transaction.
+  - If the chain state precedes the Eclipse upgrade, `batcherHash`, `overhead` and `scalar` are
```
Comment (tynes) [inline specs/system_config.md]: still says eclipse

Comment (roberto-bayardo) [reply specs/system_config.md]: fixed

File: specs/system_config.md (hunk 2 above)
Comment (roberto-bayardo) [reply specs/system_config.md]: fixed

**Review by tynes — APPROVED** [review]

**Comment by tynes** [comment]
This needs a lint then is good to merge

**Comment by roberto-bayardo** [comment]
> This needs a lint then is good to merge

had to change up the section heading a bit because the TOC generator was generating a long line that would then trigger the linter failure.  but should be good now.

**Pull request merged**